sebs/experiments/invocation_overhead.py

```
@@ -38,24 +53,55 @@ def __init__(self, deployment_client: FaaSSystem, benchmark: Benchmark, settings
+
+        Calculates target points for payload sizes based on experiment settings.
+
+        :param settings: Dictionary of experiment settings, expected to contain:
+                         'payload_begin', 'payload_end', 'payload_points'.
+        """
         from numpy import linspace
 
         points = linspace(
             settings["payload_begin"],
-            settings["payload_end"],
+            settings_["payload_end"],
```

Ruff (0.11.9)

Undefined name `settings_`

(F821)